doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+us to answer questions such as:
+- How does a prediction change dependent on feature inputs?
+- What features are Important for a given prediction to hold?
+- What features are not important for a given prediction to hold?
+- What set of features would you have to minimally change to obtain a new prediction of your choosing?
+- How does each feature contribute to a model's prediction?
+
+The set of insights available are dependent on the trained model. For instance, If the model is a regression it makes sense to ask how the prediction varies for some feature. Whereas, it doesn't make sense to ask what minimal change is required to obtain a new classification. Insights are constrained by:
+
+- The type of data the model handles. Some insights only apply to image data others only to textual data
+- The task the model performs. The two types of model tasks alibi handles are regression and classification
+- The type of model used. Examples of model types include neural networks and random forests
+
+Some explainer methods apply only to specific types of models such as TreeSHAP which can only be used with tree-based models. This is the case when an explainer method uses some aspect of that model's structure. If the model is a neural network then some methods require computing the prediction derivative for model inputs. Methods that require access to the model internals like this are known as **white box** methods. Other explainers apply to any type of model. They can do so because the underlying method doesn't make use of the model internals. Instead, they only need to have access to the model outputs given particular inputs. Methods that apply in this general setting are known as **black box** methods. Note that white box methods are a subset of black-box methods and an explainer being a white box method is a much stronger constraint than it being a black box method. Typically, white box methods are faster than black-box methods as access to the model internals means the method can exploit some aspect of that model's structure.
+
+
+:::{admonition} **Note 1: Black Box Definition**
+The use of Black Box here varies subtly from the conventional use within machine learning. Typically, we say a model is a black box if the mechanism by which it makes predictions is too complicated to be interpretable to a human. Here we use black-box to mean that the explainer method doesn't need access to the model internals to be applied.
+:::
+
+## Applications:
+
+**Trust:** At a core level, explainability builds trust in the machine learning systems we use. It allows us to justify their use in many contexts where an understanding of the basis of the decision is paramount.
+
+**Testing:** Explainability can be thought of as an extra form of testing for a model. The insights derived should conform to the expected behaviour. Failure to do so may indicate issues with the model or problems with the dataset it's been trained on.
```

Comment:
"The insights derived should conform to the expected behaviour" - feels like a loaded sentence, especially with Note 2 below on biases. Maybe we need to tone this down or rephrase so it's not taken the wrong way.

Reply:
Similar comment as in the note around biases below: best to showcase with a simple example, e.g. using an explainer to pick up on undesired model biases such as discovering that a key feature in terms of feature importance for a loan application might be the postcode which actually is a proxy for maybe the ethnicity or something similar. This comment is also a more general rule of thumb for me: if you can clarify things by adding a super simple example in 1 or 2 lines, it probably doesn't hurt given that the target audience are practitioners.